aztec/src/main/kotlin/org/wordpress/aztec/formatting/MediaFormatter.kt

```diff
@@ -0,0 +1,159 @@
+package org.wordpress.aztec.formatting
+
+import android.graphics.drawable.Drawable
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import androidx.appcompat.content.res.AppCompatResources
+import org.wordpress.aztec.AztecAttributes
+import org.wordpress.aztec.AztecText
+import org.wordpress.aztec.Constants
+import org.wordpress.aztec.R
+import org.wordpress.aztec.spans.AztecHorizontalRuleSpan
+import org.wordpress.aztec.spans.AztecImageSpan
+import org.wordpress.aztec.spans.AztecMediaClickableSpan
+import org.wordpress.aztec.spans.AztecMediaSpan
+import org.wordpress.aztec.spans.AztecVideoSpan
+import org.wordpress.aztec.spans.IAztecBlockSpan
+import org.wordpress.aztec.spans.IAztecNestable
+import org.wordpress.aztec.watchers.EndOfBufferMarkerAdder
+import org.xml.sax.Attributes
+
+/**
+ * This class contains methods to add media and horizontal rule to the editor.
+ */
+class MediaFormatter(editor: AztecText) : AztecFormatter(editor) {
```

Comment:
I originally created new methods in `BlockFormatter` because this change makes the line blocks behave more like regular blocks. After that I thought that the logic would be better in the same file so I created a new one. I agree that I can use the `LineBlockFormatter`. I'll move the code there 👍